metadata/packagesruntime.json

```diff
@@ -2020,7 +2020,484 @@
         "command": "./run-kotor1.sh",
         "engine_name": "reone"
     },
-    "39510": {
+    "34270": {
+        "game_name": "SEGA Mega Drive & Genesis Classics",
+        "download": [
+            {
+                "name": "BlastEm",
```

Comment:
This name is going to end up being "blastem", otherwise you'll have an invalid download and an additional one added to the array. Another option is to have an empty download array instead.